roles/validations/README.md

```diff
@@ -0,0 +1,34 @@
+# validations
+The Validations role provides a framework for performing state verification.
+This is useful for QE purposes and will be leveraged post-deployment to assert
+that required conditions have been satisfied. For example, if I update a certain
+value in my `OpenStackDataPlaneNodeSet`. I want to validate that the change has
+been applied as per my expectations on the relevant OpenStack Compute nodes.
+
+## Privilege escalation
+Privilege escalation will be dependent on the specific job being executed. Some jobs
+may require privilege escalation. Check with the individual jobs to determine
+requirements.
+
+## Parameters
+The parameters will ultimately depend on what is implemented in each validation task.
+But in order to launch this role, at a minimum the following should be defined:
+
+# If you want to specify a specific list of jobs to execute, populate The
```

Comment:
so here you want to use 3 `#` - it should be nested under the Parameters. Though you may consider using the usual way to display parameters? such as https://github.com/openstack-k8s-operators/ci-framework/blob/e64341bb5e44e42d418fa98640c61b6c654d751b/roles/artifacts/README.md?plain=1#L7-L15